test/wdio/ts-target-props/components.d.ts

```diff
@@ -38,20 +27,9 @@ declare global {
 declare namespace LocalJSX {
     interface TsTargetProps {
         "basicProp"?: string;
-        /**
-         * @deprecated use camelCase instead. Support for dash-casing will be removed in Stencil v5.
-         */
-        "basic-prop"?: string;
         "decoratedGetterSetterProp"?: number;
-        /**
-         * @deprecated use camelCase instead. Support for dash-casing will be removed in Stencil v5.
-         */
-        "decorated-getter-setter-prop"?: number;
         "decoratedProp"?: number;
-        /**
-         * @deprecated use camelCase instead. Support for dash-casing will be removed in Stencil v5.
-         */
-        "decorated-prop"?: number;
```

Comment:
I've added these deprecation warnings in https://github.com/ionic-team/stencil/pull/6158 .. I would suggest to revert these changes as the next test run will re-introduce them again. Wdyt?